http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/Boot.scala

```diff
@@ -91,6 +82,21 @@ object Boot extends IOApp {
         )
       )
 
+    val livelinessRoutes = new LivelinessRoutes
+
+    val liveliness = logger.info("Liveliness server has been created, starting...").unsafeToFuture()(cats.effect.unsafe.IORuntime.global) >> Http()
+      .newServerAt("0.0.0.0", 9000)
+      .bindFlow(livelinessRoutes.route)
+      .onError { case t: Throwable =>
+        logger
+          .error(t)("FATAL - failure starting liveliness http server")
+          .unsafeToFuture()(cats.effect.unsafe.IORuntime.global)
+      }
+
+    Await.ready(liveliness, Duration.Inf)
```

Comment:
I don't think we need ot await here...we can just let it start asynchrounously

Reply:
I guess it doesn't really matter, but the thought is it would be awkward if readiness was up before liveliness due to async thread scheduling nonsense...

We can remove it though and see

Reply:
that shouldn't happen...the `liveness` app is really simple, the start up should almost be instant...if liveness takes that long to start up, I think we will have a problem regardless